bin/push_docker.js

```diff
@@ -36,33 +36,36 @@ require('dotenv').config();
  * yarn docker staging '2021-08-25' '1240cfda9e609470cf1154e18f5bc582ca8907ff'
  */
 
-/** Either "staging" (for internal releases / staging bumps) or "production" (for cloud releases) */
-const stageParam = process.argv[2] || '';
-/** Version tag of webapp (i.e. "2021-08-25") */
-const versionParam = process.argv[3] || '';
+/** Key to  ./app-config/package.json, either
+ *
+ *  "wire-web-config-default-master"
+ *   or
+ *  "wire-web-config-default-staging"
+ * */
+const configurationEntry = process.argv[2];
+/** Version tag of webapp (e.g. "2023-11-09-staging.0", "dev") */
+const versionTag = process.argv[3];
+const uniqueTagOut = process.argv[4] || '';
```

Comment:
we also use this command in `create_docker_image` action. Let's make sure it's not going to break this workflow (I think it will not)

Reply:
hummm it might actually break this other action (since the new input is at the first place)

Reply:
Fixed in ef506e860fed7a4e33406c2f304a150563a86005